app/services/cocina/to_datacite/form.rb

```diff
@@ -19,10 +19,10 @@ def initialize(cocina_desc)
       def type_attributes
         return unless resource_type_general || resource_type
 
-        {
-          resourceTypeGeneral: resource_type_general,
-          resourceType: resource_type
-        }
+        {}.tap do |types_attributes|
+          types_attributes[:resourceTypeGeneral] = resource_type_general
+          types_attributes[:resourceType] = resource_type || ''
+        end
```

Comment:
Adding the `|| ''` means we don't need the `tap` jazz now, I think?

```suggestion
        {
          resourceTypeGeneral: resource_type_general,
          resourceType: resource_type || ''
        end
```

Reply:
Probably, I can test that.